include/ofi_list.h

```diff
@@ -219,6 +243,15 @@ static inline void dlist_splice_head(struct dlist_entry *head,
 	head->next = to_splice->next;
 	head->next->prev = head;
 
+	OFI_DBG_ADD(head->size, to_splice->size);
+#if ENABLE_DEBUG
+	struct dlist_entry *item;
+	dlist_foreach(head, item) {
+		item->list = head->list;
+		item->size = head->size;
```

Comment:
Setting size for an item doesn't seem needed and results in an inconsistent value of size for the elements on the list: some will be 0, others will have a value.

Reply:
Thank you, fixed!